nokia/sonic-platform
# Add support for H3 and update compile rules

## Why I did it
Add new Nokia build target and establish an arm64 build:
    Platform: x86_64-nokia_ixr7220_h3-r0
    HwSKU: Nokia-IXR7220-H3
    ASIC: Broadcom
    Port Config: 32x400G + 2x10G
## How I did it

1. Add platform related directory ixr7220h3 in platform/Broadcom/sonic-platform-modules-nokia
2. Modified related build scripts in platform/Broadcom/sonic-platform-modules-nokia/debian

## How to verify it

1. Make sure the sonic-buildimage is successful 
2. Run this image on x86_64-nokia_ixr7220_h3-r0 and verify all dockers are up and test basic commands like:

    o	show version
    o	show platform summary
    o	show platform syseeprom
    o	show platform fan
    o	show platform psustatus
    o	show platform firmware status
    o	show platform temperature
    o	sudo show system-health detail
    o	show interface status

1. Run OC test, for T0/T1 topology and have 100% pass on platform.
2. Run this image on x86_64-nokia_ixr7250e_sup-r0 and x86_64-nokia_ixr7250e_36x400g-r0, make sure all functions run correctly.
